.wokeignore

```diff
@@ -0,0 +1 @@
+tests/
```

Comment:
Why do we ignore all tests ?

Reply:
The issue was with the word "master" that is used in the test configuration of bind. I thought that it would be so common that we could ignore the tests

Reply:
Updated